scripts/update-tasks.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+declare -r SCRIPT_PATH=$(readlink -f "$0")
```

Comment:
`readlink -f` fails on *BSD (osx) :(((((

Reply:
thank you! I have a fix for that